source/release-notes/backend.txt

```diff
@@ -13,6 +13,35 @@ Atlas App Services Changelog
 2023 Releases
 -------------
 
+.. _backend_20231213:
+
+13 December 2023 Release
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Added support for passing in user's new metadata from a request using JWT authentication to the function context.
+
+- Apps, users, and refresh token are now pruned at the 30 day mark.
+
+- Fixed webpack production yarn build warnings, such as "asset size limit" and "entrypoint size warnings", by reducing bundle size. 
+
+- Updated App Services schema UI to display the difference between a 64-bit ``long`` and 32-bit ``integer``.
+
+- Disallowed creating schemas with invalid database or collection names, such as names including spaces.
+
+- Left default for ``allowDiskUse`` as true unless user specifies it as false. 
+
+- Fixed bug where a read document filter set to false is not unioned with a write filter.
+
+- Created a private endpoint for enabling tiered sync.
+
+- Added ``integer`` as a supported type for indexed queryable fields.
+
+- Updated the merge window filtering to use indexed queryable fields if possible.
+
+- Distributed translators evenly within the sync pool.
+
+- If one of several sync translators has failed, the failed subscription banner will now show up on the sync page.
```

Comment:
```suggestion
- Added ``integer`` as a supported type for indexed queryable fields.
```

These last three are pretty internal focused so let's remove them.

"merge window" is an implementation detail of operational transformation and is not a term we expose to the end users, so it doesn't really make much sense here. Also, the potential bug it fixes is very rare so I think we can just leave it out.

Similar idea with the other two - our k8s configuration and management of sync translators is something we abstract away from the end user.